src/System.Configuration.ConfigurationManager/tests/Mono/TestUtil.cs

```diff
@@ -67,8 +67,7 @@ public static string ThisConfigFileName
         {
             get
             {
-                var exe = Path.GetFileName(ThisApplicationPath);
-                return exe + ".config";
+                return Assembly.GetEntryAssembly().GetName().Name + ".config";
```

Comment:
I need to see why ThisApplicationPath is different on AOT and core.Changing it to ".exe.config" fails in AOT.